packages/cli/src/commands/cliRequestSwapDepositAddress.ts

```diff
@@ -69,6 +69,11 @@ export default async function cliRequestSwapDepositAddress(
       ccmParams,
       commissionBps: args.commission ?? 0,
       affiliates: [],
+      fillOrKillParams: {
```

Comment:
this command is not used by anybody, so well just add some dummy data to satisfy typescript for now